node/test/go/dev-mode/test/integration/index.test.js

```diff
@@ -56,15 +56,21 @@ describe('Integration', function () {
         ]),
         config: {
           test: ({ invocationsData }) => {
-            for (const [, reqRes] of invocationsData.map((data) => data.reqRes).entries()) {
-              expect(reqRes.length).to.equal(1);
-              for (const payload of reqRes) {
-                expect(payload).to.haveOwnProperty('region');
-                expect(payload).to.haveOwnProperty('accountId');
-                expect(payload).to.haveOwnProperty('timestamps');
-                expect(payload).to.haveOwnProperty('payloads');
-              }
-            }
+            // Replace with external + sdk integration results once the sdk is configured
```

Comment:
Is the expectation that once we send logs to an external extension, we stop writing payloads to logs?

Reply:
If I am understanding the question correctly, I don't think so. I believe we still want to write the existing sdk payloads to cloudwatch whether or not we are sending data to the external extension.